### DISCUSSION TITLE
[v4] Adonis WebSockets Client - How to remove log statements in Nuxt.js

### DISCUSSION THREAD

**Discussion by kempsteven** [adonisjs/core#1728] [Help] [answered]
I saw [here](https://adonisjs.com/docs/4.1/websocket-client#_production_builds) that I should defining `NODE_ENV` via `Webpack DefinePlugin` or `rollup-plugin-replace`

`"@adonisjs/websocket-client": "^1.0.9"`

and I did that but nothing changed, when I build the nuxt project and then `nuxt start`

here is my nuxt.config.js

```js
import colors from 'vuetify/es5/util/colors'
require('dotenv').config()
import webpack from 'webpack'

export default {
  ...

  build: {
   ...

   plugins: [
      new webpack.DefinePlugin({
        'NODE_ENV': 'production'
      })
    ]
  }
  ...
}

```

**Comment by kempsteven** [comment]
Anyone pls? :(

**Reply by thetutlage** [reply]
This line of code checks for the `NODE_ENV` https://github.com/adonisjs/adonis-websocket-client/blob/develop/src/Debug/index.js#L12. Can you log the `NODE_ENV` in your app to see if it is actually set to `production`?

**Reply by kempsteven** [reply]
Yes its shows that it is in `production`, anyways, thank you for responding, I've fixed the issue using what this guy [did](https://github.com/adonisjs/adonis-websocket-client/issues/50#issuecomment-480640515)

**Comment by kempsteven** [comment, marked as answer]
The fix to this is to add a `/index` when importing the package in a component, which is kinda weird but cant see any solution to this.

**It would look like this now:**
`import Ws from '@adonisjs/websocket-client/index'`
